A2 - Loops & Literals/first_efficient.rb

```diff
@@ -0,0 +1,30 @@
+# Returns sum of values in list
+def sum(list)
+  list.reduce(&:+).to_i
+end
+
+def valid?(list)
+  right_sum = sum(list) - list[0]
+  left_sum = 0
+  (list.length).times do |index|
```

Comment:
instead of 
```
(list.length).times do |index|
... list[index] ..
end
```

what if we do this way
```
list.each do |list_element|

end
```